scala/scalafmt/BUILD

```diff
@@ -0,0 +1,37 @@
+load("//scala:scala.bzl", "scala_binary")
+
+filegroup(
+    name = "runner",
+    srcs = ["private/format.template.sh"],
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "testrunner",
+    srcs = ["private/format-test.template.sh"],
+    visibility = ["//visibility:public"],
+)
+
+scala_binary(
+    name = "scalafmt",
+    srcs = glob(["scalafmt/ScalafmtRunner.scala"]),
+    main_class = "io.bazel.rules_scala.scalafmt.ScalafmtRunner",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//external:io_bazel_rules_scala/scalafmt/argparse4j",
+        "//external:io_bazel_rules_scala/scalafmt/geirsson_metaconfig_core",
+        "//external:io_bazel_rules_scala/scalafmt/scalafmt_parsers",
+        "//src/java/io/bazel/rulesscala/worker",
+        "@scalafmt//:org_scalameta_scalafmt_core_2_11",
+    ],
+)
+
+load(
+    "//scala/scalafmt:phase_scalafmt_ext.bzl",
+    "scalafmt_singleton",
+)
+
+scalafmt_singleton(
```

Comment:
`scalafmt_standalone`? what does it mean it's singleton?

Reply:
how is this used?

Reply:
ok, now I saw how this is used (and re-read the docs). I agree we need naming convention here.
I think we need it for the bzl filename like you say, for the extension rule and extension target. I'm also not sure if it's a good idea that it will be named like the implementation function (at least this is how the [docs](https://github.com/bazelbuild/rules_scala/blob/master/docs/customizable_phase.md) show it).

Reply:
Agree on setting up a naming convention before more PRs are coming. If it's not affecting performance, we could address it soon after this is merged.

Reply:
Sure thing. Let’s do that